src/index.html

```diff
@@ -5,8 +5,403 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="navigation">
+        <a href="#home" class="logo">
+          <img src="images/logo.png" alt="logo">
+        </a>
+
+        <nav class="nav">
+          <ul class="menu">
+            <li class="menu__category">
+              <a class="menu__menuLink is-active" href="#">Apple</a>
+            </li>
+
+            <li class="menu__category">
+              <a class="menu__menuLink" href="#">Samsung</a>
+            </li>
+
+            <li class="menu__category">
+              <a class="menu__menuLink" href="#">Смартфоны и телефоны</a>
+            </li>
+
+            <li class="menu__category">
+              <a
+                class="menu__menuLink"
+                data-qa="nav-hover"
+                href="#"
+              >Ноутбуки и компьютеры</a>
+            </li>
+
+            <li class="menu__category">
+              <a class="menu__menuLink" href="#">Гаджеты</a>
+            </li>
+
+            <li class="menu__category">
+              <a class="menu__menuLink" href="#">Планшеты</a>
+            </li>
+
+            <li class="menu__category">
+              <a class="menu__menuLink" href="#">Фото</a>
+            </li>
+
+            <li class="menu__category">
+              <a class="menu__menuLink" href="#">Видео</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="content">
+      <div class="catalog">
+        <div class="product-card catalog__product-card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="product-image"
+            class="product-card__image"
+          >
+
+          <div class="product-card__product-name">
+            <p>Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+          </div>
+
+          <span class="product-card__product-code">
+            Код товара: 195434
+          </span>
+
+          <div class="product-rate">
```

Comment:
Don't need to create a new bem block. It should be a card element

Reply:
done